src/Test/ProxyTestCase.php

```diff
@@ -1,200 +0,0 @@
-<?php
-
-/*
- * This file is part of the FOSHttpCache package.
- *
- * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace FOS\HttpCache\Test;
-
-use FOS\HttpCache\Test\PHPUnit\IsCacheHitConstraint;
-use FOS\HttpCache\Test\PHPUnit\IsCacheMissConstraint;
-use Http\Adapter\HttpAdapter;
-use Http\Discovery\HttpAdapterDiscovery;
-use Http\Discovery\MessageFactoryDiscovery;
-use Http\Discovery\UriFactoryDiscovery;
-use Psr\Http\Message\RequestInterface;
-use Psr\Http\Message\ResponseInterface;
-use Psr\Http\Message\UriInterface;
-
-/**
- * Abstract caching proxy test case
- *
- */
-abstract class ProxyTestCase extends \PHPUnit_Framework_TestCase
```

Comment:
removing this class is a BC break if someone was extending it directly when writing their tests